fetcher/tests/test_fetch.py

```diff
@@ -220,15 +221,23 @@ def tearDown(self):
         self.ctx.close()
         super().tearDown()
 
-    def _err(self, message: str) -> FetchResult:
+    def _err(self, message: Union[str, I18nMessage]) -> FetchResult:
```

Comment:
We only ever pass `I18nMessage`, so please nix `str` support.

In retrospect, I shouldn't have created this helper function. Helper functions in test suites are so hard to maintain.